Explain FETCH routing for relays (#1117)
moq-wg/moq-transport: draft-ietf-moq-transport.md

Relays MUST forward SUBSCRIBE messages to all publishers and PUBLISH_NAMESPACE
and PUBLISH messages to all subscribers that have a Namespace Prefix Match.

When a Relay needs to make an upstream FETCH request, it determines the

> available publishers using the same matching rules as SUBSCRIBE. When more than

Author: I think we should discuss whether prefix length is a determinant separately from this PR - can you file a separate issue?

> When a Relay needs to make an upstream FETCH request, it determines the
> available publishers using the same matching rules as SUBSCRIBE. When more than
> one publisher is available, the Relay MUST send the FETCH to at least one of
> them.  If FETCH fails with `UNKNOWN_STATUS_IN_RANGE` (see {{message-fetch}}),
> the Relay MUST attempt the FETCH against other matching publishers until the
> FETCH succeeds or fails for a different reason.
> 

Author: ```suggestion
When a Relay needs to make an upstream FETCH request, it determines the
available publishers using the same matching rules as SUBSCRIBE. When more than
one publisher is available, the Relay MAY send the FETCH to any of them.

```
